w08h01/test/pgdp/SimpleGenericsTest.java

```diff
@@ -141,6 +141,22 @@ void intersection2() {
         Collection<ArrayList>[] listArray = new Collection[]{list1, list, list3};
         assertEquals("[1, 2, 4]", Arrays.toString(SimpleGenerics.intersection(listArray).toArray()));
     }
+    @Test
```

Comment:
```suggestion

    @Test
```

Leave an empty between the tests